src/Counter.elm

```diff
@@ -15,21 +15,21 @@ type Msg
 
 init : Model
 init =
-    Model 0
+    0
 
 
 update : Msg -> Model -> Model
 update msg model =
     case msg of
         Plus ->
-            { model | counter = model.counter + 1 }
+            model + 1
 
         Minus ->
-            if model.counter == 0 then
+            if model == 0 then
                 model
 
             else
-                { model | counter = model.counter - 1 }
+                model - 1
```

Comment:
Can you do this without an `if`? There's a useful, rather common math function you can use.